docker/README.md

```diff
@@ -0,0 +1,46 @@
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+This module contains a number of Docker images to ease testing of
+Avatica clients against a known-server.
+
+## Docker
+
+`src/main/docker` contains a number of Docker images and Docker-compose
+configuration files to launch a standalone-Avatica server. Maven automation
+exists for the base Docker image "avatica-server" which can be invoked with
+the "-Pdocker" Maven profile.
+
+The other Docker images must be built by hand.
+
+### Provided Images
+
+A number of Docker images for different databases are provided. Presently, they include:
+
+* [HyperSQL](https://github.com/apache/calcite-avatica/tree/master/docker/src/main/docker/hypersql)
+* [MySQL](https://github.com/apache/calcite-avatica/tree/master/docker/src/main/docker/mysql)
+* [PostgreSQL(https://github.com/apache/calcite-avatica/tree/master/docker/src/main/docker/postgresql)
+
+## Dockerhub
+
+`src/main/dockerhub` contains a copy of the same `avatica-server` Dockerfile
```

Comment:
Yeah, sadly. This is "state-of-the-art" as I understand it. We have the ability to build the "snapshot" image using spotify's maven-docker-plugin, but this doesn't work for the ASF release process (in which we wanna pull from the "officially release artifacts").